applications/nrf5340_audio/src/bluetooth/le_audio_bis_headset.c

```diff
@@ -4,3 +4,240 @@
  * SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
  */
 
+#if (CONFIG_AUDIO_DEV == HEADSET)
+
+#include <bluetooth/bluetooth.h>
+#include <bluetooth/audio/audio.h>
+
+#include "le_audio.h"
+#include "macros_common.h"
+#include "ctrl_events.h"
+
+#include <logging/log.h>
+LOG_MODULE_REGISTER(bis_headset, CONFIG_LOG_BLE_LEVEL);
+
+// TODO: Implement timeouts for the events (can use same timeout functionality for every step)
+
+static le_audio_receive_cb receive_cb;
+static bool synced_to_broadcast;
+
+static struct bt_audio_broadcast_sink *broadcast_sink;
+// TODO: Currently only 1 stream is supported
+static struct bt_audio_stream streams[CONFIG_BT_AUDIO_BROADCAST_SNK_STREAM_COUNT];
+
+/* Mandatory support preset for both source and sink */
+static struct bt_audio_lc3_preset lc3_preset = BT_AUDIO_LC3_BROADCAST_PRESET_48_4_1;
+
+/* Create a mask for the maximum BIS we can sync to using the number of streams
+ * we have. We add an additional 1 since the bis indexes start from 1 and not
+ * 0.
+ */
+static const uint32_t bis_index_mask = BIT_MASK(ARRAY_SIZE(streams) + 1U);
+static uint32_t bis_index_bitfield;
+
+static void stream_started_cb(struct bt_audio_stream *stream)
+{
+	int ret;
+	struct event_t event;
+
+	event.event_source = EVT_SRC_LE_AUDIO;
+	event.le_audio_activity.le_audio_evt_type = LE_AUDIO_EVT_STREAMING;
+	
+	ret = ctrl_events_put(&event);
+	ERR_CHK(ret);
+	
+	LOG_INF("Stream started");
+}
+
+static void stream_stopped_cb(struct bt_audio_stream *stream)
+{
+	LOG_INF("Stream stopped");
```

Comment:
Create LE_AUDIO_EVT_NOT_STREAMING event, to stop the codec(s)

Reply:
Done